pom.xml

```diff
@@ -950,6 +964,20 @@
                 </plugin>
 
                 <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-dependency-plugin</artifactId>
+                    <!--
+                    Annotations are apparently the canonical use for this:
+                    http://maven.apache.org/components/plugins/maven-dependency-plugin/examples/exclude-dependencies-from-dependency-analysis.html
+                    -->
+                    <configuration>
```

Comment:
I don't think globally ignoring this is the best way. I'd like to see where this is needed, since it doesn't seem to be declared anywhere as a dependency.

Marking the dependency as `<scope>provided</scope>` at the usage point is probably better.